src/Routes/RosPage/RosPage.js

```diff
@@ -178,11 +205,21 @@ class RosPage extends React.Component {
         });
     }
 
+    updateOsFilter = (value) => {
```

Comment:
Nitpick: Can we keep the `OS` as capital her as we are following that in the component? 

```suggestion
    updateOSFilter = (value) => {
```

Reply:
Done!